app/containers/message/Urls.tsx

```diff
@@ -127,7 +127,7 @@ const Url = ({ url }: { url: IUrl }) => {
 	const { colors, theme } = useTheme();
 	const { baseUrl, user } = useContext(MessageContext);
 	let image = url.image || url.url;
-	image = image.includes('http') ? image : `${baseUrl}/${image}?rc_uid=${user.id}&rc_token=${user.token}`;
+	image = image?.includes('http') ? image : `${baseUrl}/${image}?rc_uid=${user.id}&rc_token=${user.token}`;
```

Comment:
If `image` doesn't have `includes`, it means it's null or undefined.
It's being used on the else of the iif, so it's still creating something like `https://open.rocket.chat/undefined...`
You got rid of the error, but it's still returning a wrong image.
If there's no image, you should not display it at all.